### DISCUSSION TITLE
How would I go about accessing (and backing up) the Plausible databases?

### DISCUSSION THREAD

**Discussion by connervieira** [plausible/analytics#1083] [Self-Hosted Support] [answered]
In short: Is there a file somewhere containing the Plausible databases that I can view and backup?

I've been using Plausible Analytics for a while, and it's been working fantastically for my website. However, I've always wondered how to access the raw databases created and used by Plausible. I'd love to be able to programmatically analyze them, as well as back them up. I use a utility that backs up a specified list of file paths, but I'm not sure where the Plausible databases are stored, and none of the documentation I've found have seemed to help.

I have Plausible installed via Docker.

**Comment by ukutaht** [comment, marked as answer]
Hey!

The question about accessing the databases is answered here: https://github.com/plausible/analytics/discussions/621
